docs/src/use-cases.rst

```diff
@@ -360,7 +360,7 @@ So, first off, let's start by training a classification model using the followin
 The above configuration essentially means that we will be training a ResNet model with
 default settings on CIFAR10 using all 10 classes. You can launch the training process via::
 
-    $ thelper new <PATH_TO_CLASSIF_CIFAR10_CONFIG>.json <PATH_TO_OUTPUT_DIR>
+    $ thelper new -c <PATH_TO_CLASSIF_CIFAR10_CONFIG>.json -d <PATH_TO_OUTPUT_DIR>
```

Comment:
Good catch on these, there's probably a ton of other dead CLI examples due to the commandline changes, I should go around and fix them...